config/proguard/firebase-components-proguard-rules.pro

```diff
@@ -1,6 +1,8 @@
 # Reference: https://github.com/firebase/firebase-android-sdk/blob/82b02af331/firebase-components/proguard.txt.
+# Reference: https://github.com/firebase/firebase-android-sdk/blob/master/firebase-firestore/proguard.txt.
```

Comment:
Just a note -- what is on "master" could change. You might want to pin a specific commit as was done in the previous line.

Reply:
Thanks for catching this @seanlip, I have updated the url